tap_dynamics/client.py

```diff
@@ -207,7 +224,22 @@ def _make_request(self, method, endpoint, paging=False, headers=None, params=Non
         return results
 
     def get(self, endpoint, paging=False, headers=None, params=None):
-        return self._make_request("GET", endpoint, paging, headers=headers, params=params)
+        try:
+            return self._make_request("GET", endpoint, paging, headers=headers, params=params)
+        except DynamicsException as e:
+            message, error_code = None, None
+            try:
+                error_json = e.response.json()
+                message = error_json["error"]["message"]
+                error_code = error_json["error"]["code"]
+            except:
+                pass
+
+            if message is not None and error_code is not None:
+                raise SymonException(f'Import failed with the following MS Dynamics error: (error code: {error_code}) {message}','dynamics.DynamicsApiError')
```

Comment:
example:
`tap_dynamics.symon_exception.SymonException: Import failed with the following MS Dynamics error: (error code: 0x80060888) EntityMetadata With Id = LogicalName='account1' does not exist.`

err doc:  https://learn.microsoft.com/en-us/power-apps/developer/data-platform/reference/web-service-error-codes
- not sure if this should be used, couldn't find matching matching error code used in the example.